### ISSUE TITLE
"Historical" Costs vs Gameplay Costs

### ISSUE THREAD

**Issue by pap1723** [KSP-RO/RP-0#945]
I have been thinking about how we have historically "priced" out items in RP-0 and what that means for the current iterations of RP-1. With the new rollout costs, tooling costs and tech tree, sticking to "historical" dollar amounts does not seem to make a lot of sense to me.

I think looking at the historical numbers as a reference point to say "Engine X" costs a lot more to develop than "Engine Y" is 100% accurate to what we should do. But the way it works now, if I was to build a recreation of any of the real rockets, it would not come close to equaling true, real historical costs. Now, don't get me wrong, I want things to be realistic overall in pricing structure. So a rocket built in 1960 should be similar to what it cost in RL 1960, but I think trying to shoehorn costs into these buckets does not work well.

So I propose this:

- We use the historical costs of things that we already have (which is a LOT of information) as the baseline and structure for all future pricing.
- But the goal of all price adjustments, and additions of parts from other packs, should be used to improve gameplay as relative costs and amounts
- The goal is still on realism, but it allows us to include a lot more items from mods that we cannot accurately have a proper historical cost for

I am very interested to hear what many of you have to say about this. I don't know how many people have done a lot of the costs and entryCosts in the past, but I believe this is something to have for a much longer conversation as we decide how to handle things moving forward.
@NathanKell @Bornholio @ec429 @m4gus88 @MikeOnTea

**Comment by m4gus88** [comment]
I think it would be helpful to collect some guidelines about how the costs of different parts changed over time. For example, the cost of instrument units of similar capabilities have become cheaper and cheaper. We could use these guides to find the ballpark for parts we don't know the real costs of, based on other parts we already have costs for.

**Comment by MikeOnTea** [comment]
I agree, use historic costs as a baseline if available but adjust where it makes sense. For example the E-1 - its historic price might've been so expensive because it was a prototype (if we do have data at all here), but we should adjust that and make it competitive with similar engines because unlike its irl-fate you can actually do design and build rockets using it, thereby getting it into serial production.

**Comment by Serina0** [comment]
Economy of scale does come into play.  One could argue that it should be possible to tool engines in order to reduce per engine cost as well. but the prices of things like engines could be modified by their performance given their time of creation with further changes depending on if it uses any particular exotic materials that are expensive. 

One thing to consider is that older tech becomes cheaper to create as "Time" passes and new techniques are researched. without improving the quality of the camera, the amount of money spent on building it would go down due to miniaturization and other factors.

**Comment by NathanKell** [comment]
The original design of the tooling system was as part of a larger
manufacturing system, yeah. Ah well.

On the topic at hand: I had always been firmly in favor of historical
costs, and spent considerable effort trying to tune the tooling system and
our (known) costs to try to provide for broadly correct LV flyaway and
launch costs. But it's a circle that can't be squared, really. So I say use
historical data when known as a baseline, but do tweak when needed.

And on the Soviet side we have next to no idea of actual costs, even if we
have historical data, so...yeah.

Things I'd look for: Thor/Delta and Atlas we have fairly good costing on,
so the per-part costs, the flyaway (KCT construction) cost, and the launch
(KCT rollout) cost should be roughly correct throughout. IIRC the E-1 had a
madeup cost because there wasn't any data on it. We *do* have fairly good
data on Apollo/Saturn, so those costs should be kept pretty close to
historical.
Also given all the USAF did on Titan costs, we should have good data there.

Everything else...dunno.

**Comment by Bornholio** [comment]
okay my two cents, nothing space related gets cheaper, in the industry you are forced to meet ever stringent testing and inspection requirements.  Much as the same hardware on a plane that can be had for a car (commonly 30x as expensive) the multiplier due to QC/QA gets vastly higher than basic manufacturing costs.  The costs for safely disposing of waste hypergolics out runs their fuel cost.   We started in the fifties with many of these costs turned off.  The processing power of an entire atlas system is available in a '80 postage stamp sized chip, but chips that run space hardware, radiation reset hardened don't develop at the rates consumer and even reset resistant commercial hardware does. this hardware is eaten up pushing capability up not cost down.

**Comment by Serina0** [comment]
I see your point, but Space X has announced that they'll only be making Sea level Raptor engines in order to ease manufacturing and cut costs.

Better manufacturing techniques leading to fewer flaws in the finished product and less things need to be replaced during testing meaning overall less costs. I don't know if that cost ever was reduced for the end customer, but it certainly meant higher profit margins for the companies producing the parts. We then need to ask if Rp-1 wants to take the: everything is done by contractors route or space x's we build our own stuff. Given how tooling is a thing and we have to pay for it, then it suggests the latter rather than the former.

**Comment by pap1723** [comment]
@Serina0 your point about SpaceX is well taken, but remember that basically everything that SpaceX is doing is an outlier to how EVERY other space agency has operated in terms of costs and efficiency. We cannot use them as a baseline for much of any discussions. Just compare their current development to the SLS (please do not start the SLS debate, we all agree with all the things said about it).

The full scale manufacturing system hasn't been officially called off, but it has mostly been called off. There is not a good way for the game to limit the things a player can do. For example, let's say that we base the manufacturing system on the exclusive use of Subassemblies. That could work great, but NOTHING stops a player (not even intending to cheat) from taking a subassembly that is built through the manufacturing system, and then completely changing it in the editor.

Trying to come up with a system that will work, be intuitive for the player and be enjoyable to use has been determined to likely be incredibly difficult. The tooling system we have now is not ideal, but it "works" well enough now to simulate it in an arbitrary way.
